cmd/backup/script.go

```diff
@@ -121,10 +121,14 @@ func newScript() (*script, error) {
 	}
 
 	if s.c.AwsS3BucketName != "" {
+		AccessKeyID, err := s.c.ResolveSecret(s.c.AwsAccessKeyID, s.c.AwsAccessKeyIDFile)
+		s.must(err)
```

Comment:
The signature here is `newScript() (*script, error)` so instead of raising a panic here it would be better to return the error instead:

```go
if err != nil {
  return nil, fmt.Errorf("newScript: error resolving secret: %w, err)
}
```

This also applies below.